bundle.yml

```diff
@@ -54,14 +54,19 @@ projects :
             ECKIT_ENABLE_ECKIT_SQL=OFF
             ECKIT_ENABLE_WARNINGS=OFF
 
-    - field_api :
-        git     : https://github.com/ecmwf-ifs/field_api
-        version : develop-ecmwf
+    - fiat      :
+        git     : https://github.com/ecmwf-ifs/fiat
+        version : 1.5.1 
         require : ecbuild
         optional: true
+            
+    - field_api :
+        git     : https://github.com/ecmwf-ifs/field_api.git
+        version : develop-ecmwf 
+        require : ecbuild fiat
         cmake   : >
             UTIL_MODULE_PATH=${CMAKE_SOURCE_DIR}/cloudsc-dwarf/src/common/module
```

Comment:
With fiat in the bundle, field_api shouldn't require the util_module_path anymore
```suggestion
```